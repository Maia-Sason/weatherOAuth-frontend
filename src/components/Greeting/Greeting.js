import { Typography, makeStyles } from "@material-ui/core";

const useStyles = makeStyles((theme) => ({
  headerA: {
    fontSize: "2em",
    color: "white",
  },
  headerABG: {
    backgroundColor: "black",
    padding: "10px",
  },
}));

const Greeting = ({ name }) => {
  const classes = useStyles();
  return (
    <>
      <Typography className={classes.headerA}>
        <span className={classes.headerABG}>Hello {name}</span>
      </Typography>
    </>
  );
};

export default Greeting;
