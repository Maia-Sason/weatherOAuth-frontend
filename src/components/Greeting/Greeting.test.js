import React from "react";
import ReactDOM from "react-dom";
import { render } from "@testing-library/react";
import Greeting from "./Greeting";

test("<Greeting /> matches snapshot", () => {
  const component = render(<Greeting name={"Billy"} />);
  expect(component.container).toMatchSnapshot();
});

it("renders Greeting without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Greeting name={"Billy"} />, div);
});
