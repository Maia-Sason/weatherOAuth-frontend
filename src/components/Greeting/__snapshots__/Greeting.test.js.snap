// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Greeting /> matches snapshot 1`] = `
<div>
  <p
    class="MuiTypography-root makeStyles-headerA-1 MuiTypography-body1"
  >
    <span
      class="makeStyles-headerABG-2"
    >
      Hello 
      Billy
    </span>
  </p>
</div>
`;
